src/plugins/profiling/public/components/subchart.tsx

```diff
@@ -8,28 +8,46 @@
 
 import React, { useEffect } from 'react';
 
-import { Axis, BarSeries, Chart, Settings } from '@elastic/charts';
+import {
+  AreaSeries,
+  Axis,
+  Chart,
+  CurveType,
+  ScaleType,
+  Settings,
+  timeFormatter,
+} from '@elastic/charts';
 
-import { timeFormatter } from '@elastic/charts';
+import { CountPerTime } from '../../common/topn';
 
-export interface BarChartProps {
+export interface SubChartProps {
   id: string;
   name: string;
   height: number;
-  data: any[];
+  data: CountPerTime[];
   x: string;
   y: string;
 }
 
-export const BarChart: React.FC<BarChartProps> = ({ id, name, height, data, x, y }) => {
+export const SubChart: React.FC<SubChartProps> = ({ id, name, height, data, x, y }) => {
   useEffect(() => {
-    console.log(new Date().toISOString(), 'updated bar-chart');
+    console.log(new Date().toISOString(), `updated subchart ${name}`);
   }, [id, name, height, data, x, y]);
 
   return (
     <Chart size={{ height }}>
       <Settings showLegend={false} />
-      <BarSeries id={id} name={name} data={data} xScaleType="time" xAccessor={x} yAccessors={[y]} />
+      <AreaSeries
+        id={id}
+        name={name}
+        data={data}
+        xAccessor={x}
+        yAccessors={[y]}
+        xScaleType={ScaleType.Time}
+        yScaleType={ScaleType.Linear}
+        areaSeriesStyle={{ area: { opacity: 0.3 }, line: { opacity: 1 } }}
```

Comment:
I'd prefer a "stronger" line (e.g. dark blue or green and double the current thickness).
And maybe further reduce the opacity of the area !?

Reply:
Thanks, I copied these settings from the other UI, but there's definitely some design work to be done since Elastic and D3 appear to render similar settings differently.

I think we should pause on this for now and address it when I pick up the task in https://github.com/elastic/prodfiler/issues/2362 to match the colors in the combined chart with the individual charts.